app/controllers/games_controller.rb

```diff
@@ -1,10 +1,14 @@
 class GamesController < ApplicationController
   def new
     @game = Game.new
+    @game.save
+    @game.add_white_starting_pieces!
+    @game.add_black_starting_pieces!
   end
 
   def create
     @game = Game.create(game_params)
+
```

Comment:
Right here, could do something like

`@game.populate_game!`

^ This should work, because by calling `create`, you have both initialized and saved a new game. So the ID will be present (which is the issue I think you were having before, couldn't create pieces because the ID didn't exist yet for game)

Then inside of the model,

```
#game.rb
def populate_game!
    add_starting_pieces_for_color!('black')
    add_starting_pieces_for_color!('white')
end
```